aws/resources/config_service.go

```diff
@@ -46,33 +46,43 @@ func (csr *ConfigServiceRule) nukeAll(configRuleNames []string) error {
 	var deletedConfigRuleNames []*string
 
 	for _, configRuleName := range configRuleNames {
-		params := &configservice.DeleteConfigRuleInput{
+		pterm.Debug.Println(fmt.Sprintf("Start deleting config service rule: %s", configRuleName))
+		_, err := csr.Client.DeleteRemediationConfiguration(&configservice.DeleteRemediationConfigurationInput{
 			ConfigRuleName: aws.String(configRuleName),
-		}
-		_, err := csr.Client.DeleteConfigRule(params)
+		})
+		if err != nil {
+			pterm.Error.Println(fmt.Sprintf("Failed to delete remediation configuration w/ err %s", err))
+			report.Record(report.Entry{
+				Identifier:   configRuleName,
+				ResourceType: "Config service rule",
+				Error:        err,
+			})
 
-		// Record status of this resource
-		e := report.Entry{
-			Identifier:   configRuleName,
-			ResourceType: "Config service rule",
-			Error:        err,
+			continue
 		}
-		report.Record(e)
 
+		params := &configservice.DeleteConfigRuleInput{
+			ConfigRuleName: aws.String(configRuleName),
+		}
+		_, err = csr.Client.DeleteConfigRule(params)
 		if err != nil {
-			logging.Logger.Debugf("[Failed] %s", err)
-			telemetry.TrackEvent(commonTelemetry.EventContext{
-				EventName: "Error Nuking Config Service Rule",
-			}, map[string]interface{}{
-				"region": csr.Region,
+			pterm.Error.Println(fmt.Sprintf("Failed to delete config rule w/ err %s", err))
```

Comment:
Do we need `pterm` over our existing logger for any particular reason?

Reply:
I believe we should change all the logging with `pterm` or come up with our internal logging methods so we can make changes easily in the future as well. 

The main reason to use `pterm` is because of the ui. If we don't use `pterm` logging, the formatting looks odd.